data/blog/eleventh-wave-of-nostr-grants.mdx

```diff
@@ -0,0 +1,173 @@
+[NWC](https://github.com/getAlby/nostr-wallet-connect#) and
+[Cashu](https://cashu.space/); offline bitcoin transaction support with a nostr
+relay bot; GUI server tools for global operators;
+[FLDIGI](https://sourceforge.net/p/fldigi/fldigi/ci/master/tree/README)
+integration for long-range communications;
+[Reticulum-based](https://reticulum.network/) encryption and routing; and a
+flagship radio installation in Arizona to serve as a global access point. HAMSTR
+aims to ensure decentralized communication and payments remain viable—even
+when the internet is not.
+
+Repository: [LibertyFarmer/hamstr](https://github.com/LibertyFarmer/hamstr)  
+License: GPL v3.0
+
+### Vertex
+
+Vertex provides Web of Trust infrastructure for nostr, helping client developers
+filter spam, personalize content, and preserve decentralization. The service
+continuously crawls nostr relays to map [the social
+graph](https://pippellia.com/pippellia/Social+Graph/Navigating+the+social+graph)
+and applies [PageRank-style](https://en.wikipedia.org/wiki/PageRank) algorithms
+to return relevance-ranked results through
+[DVM](https://github.com/nostr-protocol/nips/blob/master/90.md) requests. The
+project has shipped core tooling—including
+[crawler](https://github.com/vertex-lab/crawler) and
+[relay](https://github.com/vertex-lab/relay) software—and integrated its
+[VerifyReputation](https://vertexlab.io/docs/nips/verify-reputation-dvm/) DVM
+into live apps.
+
+This grant will support the development of a Web of Trust-based search DVM for
+discovering trusted profiles, a fast public key viewer with impersonator
+detection, and algorithmic improvements using mutes and reports. Future updates
+will focus on client-side verification of DVM responses and personalization
+enhancements based on user interests and metadata—all while minimizing
+infrastructure requirements for clients.
+
+Repositories: [vertex-lab](https://github.com/vertex-lab)  
+License: MIT
+
+### Secure Direct Messages
+
+Secure Direct Messages brings end-to-end encrypted private messaging to nostr
+clients by offering a library that implements the Double Ratchet algorithm. By
+protecting message metadata and securing histories even if a user's nostr key is
+compromised, this approach significantly enhances privacy across the network.
+The protocol is currently available as a TypeScript library and integrated into
+clients such as [Iris](https://iris.to/) and [Damus](https://damus.io/).
+Complementary work includes a high-performance
+[Rust/LMDB](https://crates.io/crates/nostr-lmdb) server for notifications and
+webhook subscriptions.
+
+Upcoming milestones include group messaging support, multi-device session
+management [modeled on Signal](https://signal.org/docs/specifications/sesame/),
+and the public release of Double Ratchet in Damus. The team also plans to
+finalize and standardize the protocol through [NIPs 117 and
+118](https://github.com/nostr-protocol/nips/pull/1813/commits), expanding
```

Comment:
Better to link to the NIPs on the feature branch directly, instead of the commits:

- https://github.com/mmalmi/nips/blob/double-ratchet/117.md
- https://github.com/mmalmi/nips/blob/double-ratchet/118.md